.circleci/config.yml

```diff
@@ -44,31 +44,31 @@ jobs:
             prohibit-tab-character.sh
             prohibit-trailing-whitespace.sh
 
-  tests_py36:
+  tests_py36_git_1_8_0:
```

Comment:
I've granted you admin access to this repo, pls change the required checks to pass for master/develop under https://github.com/VirtusLab/git-machete/settings/branches

Reply:
Actually! as suggested in the issue, would it be possible to use `test python 3.6, git 1.8.10` jobs names? probably some quoting would be need in yaml for that

Reply:
Unfortunately it's not possible as far as can see. It's connected to how circleCI validates jobs names: every gob must meet regex pattern: `^[A-Za-z][A-Za-z\s\d_-]*$` where dots are not allowed. But we could use something similar to `test against python 3_6 and git 1_8_10` only.